.github/workflows/ios-reusable-workflow.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install Dependencies
         run: node ./install.js
       - name: Build all ios native templates
-        run: ./test/test_force.js --exit-on-failure --cli=forceios
+        run: ./test/test_force.js --exit-on-failure --cli=forceios --spm-update
```

Comment:
NB: for spm update to work https://github.com/forcedotcom/SalesforceMobileSDK-iOS-SPM/pull/8 this needed.

Reply:
And also https://github.com/forcedotcom/SalesforceMobileSDK-iOS-SPM/pull/9